pkg/network/namescheme/networknamescheme.go

```diff
@@ -20,14 +20,19 @@
 package namescheme
 
 import (
+	"crypto/md5"
 	"fmt"
+	"io"
 
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 )
 
-const PrimaryPodInterfaceName = "eth0"
+const (
+	MaxIfaceNameLen         = 15
```

Comment:
oops, previous comment quote the wrong code snippet for, I mean this line.

Reply:
Resolving this comment. Thanks again for raising.